addons/website/controllers/main.py

```diff
@@ -350,9 +350,9 @@ def theme_customize(self, enable=None, disable=None, get_bundle=False):
         if get_bundle:
             context = dict(request.context)
             return {
-                'web.assets_common': request.env["ir.qweb"]._get_asset('web.assets_common', options=context),
-                'web.assets_frontend': request.env["ir.qweb"]._get_asset('web.assets_frontend', options=context),
-                'website.assets_editor': request.env["ir.qweb"]._get_asset('website.assets_editor', options=context),
```

Comment:
Sure about that ?

Reply:
This code was deprecated since v11, I will double check but I am sure there is a tour that test the theme customization right :hugs:?
My runbot is (almost) green :wink:

Reply:
Actually this is not correct, it breaks theme customization.

But using that method is not good and should be changed, see https://github.com/odoo/odoo/commit/df4072b0b493000ccfa4e453d785bcea27659e5c.
I think I should remove this commit from this PR and it should be fixed on another PR, probably on lower version?

And since runbot is green with theme customization broken, I'd suggest to open a task to create a tour-tests about it.. This is a too big feature to be ignored..

Reply:
Why on a lower version ? There is no bug there right ?

Reply:
I don't think there is a bug, what I said is just based on the deprecating commit, I just assumed things from its message.
Is there a plan to create tour(s) on theme customization? Or any reason not to?
Otherwise, I will remove that commit from this PR and create a task to do it in master and add a tour in the same time.

Reply:
@qsm-odoo I moved deprecated commits to https://github.com/odoo/odoo/pull/33432

And about the task to create a tour, I think we should thank about which tours are needed and create a meta task to list all missing tours. Too much feature remains untested right now. Unsplash, Theme customize, Page property..
/cc @JKE-be @sbuhl